doc/source/guide/introduction/installing.rst

```diff
@@ -2,36 +2,124 @@
 Installing
 ##########
 
-Installing odl is intended to be straightforward, and this guide is meant for new users. To install odl you need to do the following steps:
+Installing odl is intended to be straightforward, and this guide is meant for new users. For a minimal installation you should do the following steps:
 
 1. `Install Python`_
-2. `Install Git`_
-3. `Install ODL`_
-4. `Run tests`_
+2. `Install ODL`_
+3. `Run tests`_
 
 If you have done any step before, you can ofc skip it.
 
 Install Python
 ==============
-To begin with, you need a python distribution. If you are an experienced user, you can use any distribution you'd like.
+To begin with, you need a Python distribution. Any modern Python distribution should do for the core library, but some extensions require CPython.
 
 Anaconda
 --------
-If you are a python novice using Windows, we recommend that you install a full package such as Anaconda. To install Anaconda
+If you are a python novice using Windows, we recommend that you install a full package such as Anaconda. To install Anaconda, download Anaconda from `Anaconda's webpage <https://www.continuum.io/downloads>`_. Once installed, run in a console
```

Comment:
You capitalized "Python" above (good). Do so here, too. A few too many "Anaconda" occurrences here.